src/components/FirstRegistrationForm.scss

```diff
@@ -13,9 +13,18 @@
     border-radius: 4px;
     width: $formWidth;
     @include fieldsFont;
+    font-weight: bold;
+    background-color: white;
+    color: black;
+    margin: 5px 0;
+  }
+
+  .Registration-Form-Submit-Button {
+    line-height: 50px;
     background-color: $borderColor;
     color: white;
     @include fieldsFont;
-    margin-top: 20px;
+    font-weight: normal;
```

Comment:
do we need here
font-weight: normal; ??

What is the weight by default?

Reply:
If I will not specify font-weight: normal, font will bold (see button styles). So, adding normal weight used to overwrite style specified before

Reply:
Ok